File: examples/python/nl_query_example.py
import qoery

configuration = qoery.Configuration()
configuration.api_key['ApiKeyAuth'] = 'YOUR_API_KEY'

with qoery.ApiClient(configuration) as api_client:
    api = qoery.QueriesApi(api_client)
    data = api.query_nl_get(
        "population of France",
        num_results=10,
        paragraph_extraction=False,
        plot2table=0,
    )
    print(data)
    

File: examples/js/nl_query_example.js
// Natural Language Query via SDK (GET)
const Qoery = require('qoery');

const client = Qoery.ApiClient.instance;
const apiKeyAuth = client.authentications['ApiKeyAuth'];
apiKeyAuth.apiKey = 'YOUR_API_KEY';

const queriesApi = new Qoery.QueriesApi();

const query = 'population of France';
const opts = {
  numResults: 10,
  paragraphExtraction: false,
  plot2table: 0
};

queriesApi.queryNlGet(query, opts, (error, data) => {
  if (error) {
    throw error;
  }
  console.log(data);
});